src/main/java/uk/gov/hmcts/reform/sscs/idam/IdamService.java

```diff
@@ -11,14 +10,18 @@
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
 
 @Service
 @Slf4j
 public class IdamService {
+
     public static final int ONE_HOUR = 1000 * 60 * 60;
 
     private final AuthTokenGenerator authTokenGenerator;
-    private final IdamApiClient idamApiClient;
+
+    private final IdamClient sharedIdamClient;
```

Comment:
probably leave it as idamClient, unless the word shared means something important.